src/node/snapshotter.h

```diff
@@ -158,8 +173,11 @@ namespace ccf
       next_snapshot_indices.push_back(last_snapshot_idx);
     }
 
-    void snapshot(consensus::Index idx)
+    void snapshot(consensus::Index idx, bool generate_snapshot)
```

Comment:
`snapshot()` doesn't seem to cover what this does, which isn't to create a snapshot most of the time. `update()` is maybe more accurate.

Reply:
Good point. Updated to `update()`.